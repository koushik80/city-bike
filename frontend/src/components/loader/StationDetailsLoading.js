import { Grid, Skeleton } from '@mui/material'
import React from 'react'

const StationDetailsLoading = () => {
  return (
    <Grid container spacing={4}>
      <Grid item xs={8} md={6} lg={6}>
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
      </Grid>
      <Grid item xs={4} md={6} lg={6}>
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '1.5rem', mt: '.5rem' }} />
      </Grid>
    </Grid>
  )
}

export default StationDetailsLoading