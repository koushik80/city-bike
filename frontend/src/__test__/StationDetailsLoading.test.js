import React from 'react'
import renderer from 'react-test-renderer'
import StationDetailsLoading from '../components/loader/StationDetailsLoading'


describe('Station Details Loading', () => {

  it('should render correctly', () => {
    const wrapper = renderer.create(<StationDetailsLoading />)

    expect(wrapper).toMatchSnapshot()
  })
})