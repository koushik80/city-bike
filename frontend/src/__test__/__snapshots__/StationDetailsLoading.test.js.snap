// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Station Details Loading should render correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 css-480o17-MuiGrid-root"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-6 css-wkfkmr-MuiGrid-root"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 MuiGrid-grid-md-6 MuiGrid-grid-lg-6 css-kdp3n0-MuiGrid-root"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1kzild6-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
  </div>
</div>
`;
